apps/system/js/notifications.js

```diff
@@ -122,6 +126,88 @@ var NotificationScreen = {
     }, function(err) {
       console.error('VersionHelper failed to lookup version settings.');
     });
+
+
+  },
+
+  getStore: function ns_getStore() {
+    return new Promise((resolve) => {
+      if (this._storeRef) {
+        return resolve(this._storeRef);
+      }
+      navigator.getDataStores(this.STORE_NAME).then((stores) => {
+        this._storeRef = stores[0];
+        return resolve(this._storeRef);
+      }, (reason) => {
+        return reject(reason);
```

Comment:
hm-m, am I missing something, or you need to specify ``reject`` as a second parameter in line 134?